solidity/contracts/bridge/Bridge.sol

```diff
@@ -1159,7 +1160,8 @@ contract Bridge is Governable, EcdsaWalletOwner {
     ///        timeout receives. The value must be in the range [0, 100]
     /// @dev Requirements:
     ///      - Moving funds transaction max total fee must be greater than zero
-    ///      - Moving funds dust threshold must be greater than zero
+    ///      - Moving funds dust threshold must be greater than zero and smaller
+    ///        than redemption dust threshold
```

Comment:
```suggestion
    ///      - Moving funds dust threshold must be greater than zero and lower
    ///        than the redemption dust threshold
```

[Here](https://english.stackexchange.com/a/26473) is a good explanation:

> Definitely use greater/less when writing in a maths context, as you say. And computing, where it's common to write and even say LT/GT (even some computer languages themselves support these forms). But outside these specialist areas it's not just 'colloquial' to use smaller - it's overwhelmingly more 'standard'.

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/241/commits/f2a5d262f26294ddcbdaf4295aa33b3cc18a5418.